.formatter.exs

```diff
@@ -0,0 +1,3 @@
+[
+    inputs: ["mix.exs", "{config,lib,test,priv}/**/*.{ex,exs}"]
```

Comment:
Thanks! :100:

Reply:
Thx, found that tip here. [hashrocket blog](https://hashrocket.com/blog/posts/format-your-elixir-code-now)